server/app/services/question/types/QuestionDefinition.java

```diff
@@ -170,17 +174,20 @@ public ImmutableSet<Locale> getSupportedLocales() {
 
   /** Get the validation predicates. */
   public final ValidationPredicates getValidationPredicates() {
-    return config.validationPredicates();
+    return config.validationPredicates().orElse(getDefaultValidationPredicates());
```

Comment:
I think we should be using orElseGet() here instead of orElse(). The orElse() is evaluated, even when having a non-empty Optional.

Reply:
Changed.